### DISCUSSION TITLE
Uniform MRI image resolution

### DISCUSSION THREAD

**Discussion by Ishak96** [facebookresearch/fastMRI#326] [Q&A] [answered]
Hello @mmuckley ,

I have a relatively simple question regarding the resolution of the K-space. Do you think it's feasible to set a uniform resolution for both the training/validation and test sets to 352 x 352 ? Would this be considered fair for comparison with state-of-the-art methods?

Thank you.

**Comment by mmuckley** [comment, marked as answer]
Hello @Ishak96, yes it should be possible from a theoretical point of view, you just need to preprocess the k-space in the right way in consideration of the Fourier transform properties, which may require some interpolation if you want to match existing resolution or fields of view.

It would not be comparable to state-of-the-art methods, since those methods are reconstructed in a standardized way. For each acquisition, the technician specifies a desired matrix size and spatial resolution, and these are encoded into the header data. So your method should be reconstructing according to those specifications, which is how all the code in the repository works. We have, in effect, done that preprocessing for you with zero-padding.

**Reply by Ishak96** [reply]
Thanks for your reply @mmuckley. As you suggested, I should rather adapt my method to deal with resolution variation.
